server/apps/publish/archive_publish.py

```diff
@@ -828,8 +863,14 @@ def get_subscribers(self, doc, target_media_type):
         """
         subscribers, subscribers_yet_to_receive, takes_subscribers = [], [], []
         first_take = None
+
         # Step 1
-        subscribers = list(get_resource_service('subscribers').get(req=None, lookup={'is_active': True}))
+        if doc.get(EMBARGO) and doc.get(EMBARGO) > utcnow():
+            subscribers = list(get_resource_service('subscribers').get(req=None,
+                                                                       lookup={'subscriber_type': SUBSCRIBER_TYPES.WIRE,
+                                                                               'is_active': True}))
+        else:
+            subscribers = list(get_resource_service('subscribers').get(req=None, lookup={'is_active': True}))
```

Comment:
It would probably be better to simply add the `subscriber_type` to the lookup dictionary if the condition holds, but otherwise keep the rest of the code in a single statement, without duplicating it across both `if` branches.